02/app.js

```diff
@@ -8,4 +8,14 @@ const calendarJS = {
     'ES7': '2016-06',
     'ES8': '2017-06',
     'ES9': '2018-06',
-}
\ No newline at end of file
+}
+
+for(const key in calendarJS){
+    console.log(key + ' wydano w terminie ' + calendarJS[key])
```

Comment:
To powinno być w `else` ponieważ zawsze występuje, a powinno tylko wtedy gdy `calendarJS[key]` nie jest `null`